brambling/templates/brambling/mail/order_receipt/body_plaintext.txt

```diff
@@ -24,7 +24,7 @@ Total: {{ total_payments|format_money:event.currency }}
 
 Order #{{ order.code }}{# Transaction date #}
 
-{% if order.person %}{% url 'brambling_event_order_summary' event_slug=event.slug organization_slug=event.organization.slug as url %}{% else %}{% url 'brambling_event_order_summary' event_slug=event.slug organization_slug=event.organization.slug code=order.code as url%}{% endif %}
+{% if order.person %}{% url 'brambling_event_order_summary' event_slug=event.slug organization_slug=event.organization.slug as url %}{% else %}{% url 'brambling_event_order_summary' event_slug=event.slug organization_slug=event.organization.slug as url%}{% endif %}
```

Comment:
this one (and the related ones in emails) actually need to direct to the newly created view, so that anonymous users end up on the page directing them to register (rather than accidentally creating a new order).